inc/Calculator.h

```diff
@@ -0,0 +1,394 @@
+#ifndef CALCUALTOR
+#define CALCULATOR
+#define RAYGUI_IMPLEMENTATION
+
+#include <string>
+#include "Widget.h"
+
+#include <raylib.h>
+#include <iostream>
+#include <ctime>
+#include "Constants.h"
+#include <stack>
+#include <list>
+#include <cmath>
+#include <iomanip>
+using namespace std;
+
+class Calculator : public Widget {
+std::string text;
+Vector2 mousePoint;
+Texture2D plusButton;
+bool Button1Pressed;
+int offset;
+int lastOperatorIndex;
+
+stack<char> operators;
+stack<float> operands;
+public:
+    Calculator(uint32_t x, uint32_t y, uint32_t w, uint32_t h, std::string n) :
+        Widget (x, y, w, h, n) {
+        // Calls widget's constructor
+        //render = false;
+        text = "";
+        mousePoint = {0.0f, 0.0f};
+        Image plusIcon = LoadImage("../resources/images/plus_operator.png");
+        ImageResize(&plusIcon, 50, 50);
+        plusButton = LoadTextureFromImage(plusIcon);
+        UnloadImage(plusIcon);
+        Button1Pressed = false;
+        offset = 25;
+        
+    }
+    void spawnButton(int x, int y, int w,int h, std::string buttonText)
+    {
```

Comment:
Be consistent with bracket notation. I say keep the curly bracket on the same line as function definition since the rest of the codebase is like that